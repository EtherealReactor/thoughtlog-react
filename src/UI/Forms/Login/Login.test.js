import React from 'react';
import { shallow } from 'enzyme';
import shallowToJson from 'enzyme-to-json';
import LoginForm from './Login';

describe("<LoginForm />", () => {
  it("must render LoginForm", () => {
    const wrapper = shallow(<LoginForm />)
    expect(wrapper).toHaveLength(1);
    expect(shallowToJson(wrapper)).toMatchSnapshot();
  });
});