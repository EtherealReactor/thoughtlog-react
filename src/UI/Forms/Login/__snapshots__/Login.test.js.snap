// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<LoginForm /> must render LoginForm 1`] = `
<Connect(Form(Login))
  destroyOnUnmount={true}
  enableReinitialize={false}
  forceUnregisterOnUnmount={false}
  form="loginForm"
  getFormState={[Function]}
  keepDirtyOnReinitialize={false}
  persistentSubmitErrors={false}
  pure={true}
  shouldAsyncValidate={[Function]}
  shouldError={[Function]}
  shouldValidate={[Function]}
  shouldWarn={[Function]}
  touchOnBlur={true}
  touchOnChange={false}
  updateUnregisteredFields={false}
/>
`;
